README.md

```diff
@@ -1,173 +1,249 @@
-# Network Observability Operator (NOO)
+# NetObserv Operator
 
-[![Docker Repository on Quay](https://quay.io/repository/netobserv/network-observability-operator/status "Docker Repository on Quay")](https://quay.io/repository/netobserv/network-observability-operator)
+A Kubernetes / OpenShift operator for network observability. It deploys a flow monitoring pipeline based on an [eBPF agent](https://github.com/netobserv/netobserv-ebpf-agent/) or [IPFIX](https://en.wikipedia.org/wiki/IP_Flow_Information_Export) exports. It provides dashboards, metrics, and keeps flows accessible in a queryable log store: [Grafana Loki](https://grafana.com/oss/loki/). When used in OpenShift, new dashboards are available in the Console.
 
```

Comment:
How about adding a screenshot here to show what the NOO can do?
May be we can reuse the topology screenshot?

Reply:
I was also thinking about showing a top banner, like on operatorhub: https://operatorhub.io/operator/netobserv-operator/
but actually, I don't know, we don't want too many screenshots either, as it takes a lot of space and forces to scroll more...

There's now 6 screenshots in this version (compared to 3 before, and they were buried at the bottom of the page), I would have said it's enough (?)

Reply:
Ok, I understand your point of not having too many screenshots.